pkg/kubelet/sysctl/runtime.go

```diff
@@ -35,15 +35,15 @@ const (
 // TODO: The admission logic in this file is runtime-dependent. It should be
 // changed to be generic and CRI-compatible.
 
-type runtimeAdmitHandler struct {
+type AdmitHandler struct {
 	result lifecycle.PodAdmitResult
 }
 
-var _ lifecycle.PodAdmitHandler = &runtimeAdmitHandler{}
+var _ lifecycle.PodAdmitHandler = &AdmitHandler{}
 
 // NewRuntimeAdmitHandler returns a sysctlRuntimeAdmitHandler which checks whether
 // the given runtime support sysctls.
-func NewRuntimeAdmitHandler(runtime container.Runtime) (*runtimeAdmitHandler, error) {
+func NewRuntimeAdmitHandler(runtime container.Runtime) (*AdmitHandler, error) {
```

Comment:
Nit: Now that we've renamed `runtimeAdmitHandler` to just `AdmitHandler`, I believe we should rename this function to `NewAdmitHandler`.